examples/vmi-fedora-isolated.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
```

Comment:
This VM needs to have a different name to not collide with `examples/vmi-fedora.yaml`.
Maybe `vmi-fedora-realtime`?

Reply:
not sure, this manifest does not have the knobs defined for realtime, maybe just `vmi-fedora-isolated`?

Reply:
sounds good to me :+1: